### PR TITLE
Add Book object and modify JSON property

### LINKED ISSUE #48: Add book object and modify JSON file

### PR DESCRIPTION
**Pull request by glyfy**
Resolves #48

### CODE REVIEW

File: src/test/java/expresslibrary/testutil/PersonBuilder.java [hunk 1]
```diff
@@ -89,6 +93,14 @@ public PersonBuilder withEmail(String email) {
...
+    /**
+     * Sets book
+     * @return
+     */
+    public PersonBuilder withBook(String title){
+        this.book = new Book(title);
+        return this;
+    }
     public Person build() {
         return new Person(name, phone, email, address, tags);
```
Comment (Jarrett0203) [inline src/test/java/expresslibrary/testutil/PersonBuilder.java:105]:
```suggestion
        return new Person(name, phone, email, address, book, tags);
```

**Review by Jarrett0203 — CHANGES_REQUESTED** [review]
Additionally please try and fix the other compilation errors.

**Review by kenzantonius — APPROVED** [review]
LGTM

**Review by Jarrett0203 — APPROVED** [review]
LGTM

**Review by kenzantonius — APPROVED** [review]
LGTM

**Pull request merged**